docs/book/quick-start.md

```diff
@@ -84,29 +84,82 @@ If you are at all uncertain about the source of a variable in your view, escape
 
 Routes determine which controller to call based on the URI and other information from the request.
 
-Configure a route and a controller in `module/Application/config/module.config.php`:
+A couple of array values are declared for routes and controllers are added to  `module/Application/config/module.config.php` as shown below:
 
 ```php
+<?php
+
+declare(strict_types=1);
+
+// Declaring our module's namespace here.  Allows us to automatically reference classes in the 
+// src folder of our module with pathnames relative to the src folder. 
+namespace Application;
+
+// Explicitly referencing these classes permits us to reference them by their names below. 
+use Laminas\Router\Http\Literal;
+use Laminas\Router\Http\Segment;
+use Laminas\ServiceManager\Factory\InvokableFactory;
+
 return [
     'router' => [
         'routes' => [
-            // Route name: used to generate links, among other things.
+            'home' => [
+                'type'    => Literal::class,
+                'options' => [
+                    'route'    => '/',
+                    'defaults' => [
+                        'controller' => Controller\IndexController::class,
+                        'action'     => 'index',
+                    ],
+                ],
+            ],
+            'application' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '/application[/:action]',
+                    'defaults' => [
+                        'controller' => Controller\IndexController::class,
+                        'action'     => 'index',
+                    ],
+                ],
+            ],
+            // New Route: used to generate links, among other things.
```

Comment:
It's not clear whether this says that the 'hello-world' route is for links, or whether routes in general help with generating links. This information can possibly be elaborated on in the first paragraph under "Create a Route".